2026-T2/MLN/assignments/Assessment3/notebook/build_v5.py

```
@@ -0,0 +1,119 @@
+
+
+def run(command):
+    print("$", " ".join(str(part) for part in command))
+    subprocess.run(command, check=True, cwd=HERE)
+
+
+def export():
+    env = {**os.environ, "JUPYTER_PATH": "/opt/homebrew/share/jupyter"}
+    EXPORT_DIR.mkdir(exist_ok=True)
+
+    subprocess.run(["jupyter", "nbconvert", "--to", "html",
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)